contracts/CO30_Matias_BlackSmokeRising.blud_cordelia/edit_me/briefing/objectives.html

```diff
@@ -0,0 +1,3 @@
+<font color='#E3D310'>PRIMARY</font>
+1- Clear locations of enemies
+2- Destroy weapon caches
```

Comment:
Space out the dashes:
1 - Clear locations of enemies
2 - Destroy weapon caches

Reply:
I'm 90% sure not spacing the dashes out is valid